### ISSUE TITLE
Transition run scripts to SLURM

### ISSUE THREAD

**Issue by akubaryk** [SWPC-IPE/WAM-IPE#275]
"Soon," Theia is moving from #PBS to #SBATCH (SLURM) directives for batch scheduling.

I'll need to create a new branch that uses `module load slurm` on Theia and verify that it functions identically to the current implementation. Migration suggestions have been made available by EMC [here](https://docs.google.com/spreadsheets/d/1h3PHJPZzlQsBqBNvG6gjIuuItUH4il1s-IJaXoAqSGA/edit#gid=0).

We'll also need to update any standalone post-processing scripts and the standalone IPE run scripts for the `ipe_refactor` branch. If there are any other feature branches that need the new job scheduler that _do not_ want to pull from development, we also need to be made aware of that.

**Comment by akubaryk** [comment]
`find . -type f | xargs grep -i -m 1 pbs` from the base WAM-IPE directory comes up with these as plausible targets for necessary changes.

```
./OldCompsetRun:  export SCHEDULER=pbs
./RT_cold_start_compsetrun:  export SCHEDULER=pbs
./oldtests/gfs_fcst_run: export SCHEDULER=pbs
./oldtests/gfs_fcst_run.IN:  elif [ $SCHEDULER = 'pbs' ]; then
./oldtests/gfs_fcst_run.IN_IBM:  elif [ $SCHEDULER = 'pbs' ]; then
./oldtests/gfs_fcst_run.IN_Linux:  elif [ $SCHEDULER = 'pbs' ]; then
./oldtests/gfs_qsub:#PBS -e /scratch3/NCEPDEV/swpc/scrub/Robert.Oehmke/rt_120236/swpc%20130316_1hr_spacewx_gsm%wam%T62_ipe%80x170/err
./oldtests/gfs_qsub.IN:#PBS -e @[RUND]/err
./oldtests/rt.sh:  export SCHEDULER=pbs
./oldtests/rt_gfs.sh:  elif [ $SCHEDULER = 'pbs' ]; then
./oldtests/rt_gfs.sh_restart:  elif [ $SCHEDULER = 'pbs' ]; then
./scripts/compsets/config/pleiades.config:## PBS stuff
./scripts/compsets/config/theia.config:## PBS stuff
./scripts/compsets/submit.sh:## set up PBS/LSF/whatever
./scripts/ipe_io_convert/qsub_theia.sh:#PBS -N ipe_convert
./scripts/plot/Convert_mpi.batch:#PBS -l procs=$TASKS
./.gitignore:#PBS/Slurm/...
./CompsetRun_New:  export SCHEDULER=pbs
```

**Comment by akubaryk** [comment]
Branch `slurm` has been created and tested (my compset scripts under standaloneWAM -- although WAM-IPE and standaloneIPE should be fine -- and Convert_mpi.batch). Going to open a pull request, so this can be closed soon.

The compset scripts proved to be very, very easy to change schedulers, which was extremely satisfying, but some of the *CompsetRun scripts and related subscripts needed more manual attention... I have added oldtests/gfs_sbatch.IN for the new scheduler, for example. I haven't tested those as I did the more commonly used scripts, as we haven't been supporting or using those older scripts in some time... if anybody still does, or knows of any other scripts in the repository that need attention, I ask that you test them on your own and report back on any issues.
